<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoDialog</class>
 <widget class="QDialog" name="VideoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/menu_icon/icon/video_icon.png</normaloff>:/menu_icon/icon/video_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#VideoDialog {
	background: #e1e1e1;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget {
	background: white;
	border-bottom: 1px solid #a1a1a1; 
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>3</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"> background: #CC181E;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Segoe UI</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding: 0px 10px;</string>
          </property>
          <property name="text">
           <string>Video Url</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding: 4px; 
border: 1px solid #aaa;
margin-right: 5px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_go">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding: 7px 14px;  
</string>
          </property>
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widget_2 {

border-top: 1px solid #ccc;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="lbl_thumbnail">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>80</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="styleSheet">
                  <string notr="true">padding: 0px 5px;</string>
                 </property>
                 <property name="text">
                  <string>Title</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="styleSheet">
                  <string notr="true">padding: 0px 5px;</string>
                 </property>
                 <property name="text">
                  <string>Quality</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="lbl_title">
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <weight>50</weight>
                   <bold>false</bold>
                   <stylestrategy>PreferAntialias</stylestrategy>
                  </font>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox">
                 <property name="minimumSize">
                  <size>
                   <width>170</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="lbl_loader">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>144</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>label</zorder>
     <zorder>verticalSpacer</zorder>
     <zorder>widget_2</zorder>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_download">
       <property name="font">
        <font>
         <family>Segoe UI</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 7px 14px; </string>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 7px 14px; </string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
